src/ca65/main.c

```diff
@@ -962,6 +975,7 @@ int main (int argc, char* argv [])
         { "--verbose",             0,      OptVerbose              },
         { "--version",             0,      OptVersion              },
         { "--warnings-as-errors",  0,      OptWarningsAsErrors     },
+        { "--expand-macros",       0,      OptExpandMacros         },
```

Comment:
The added line should go after "--debug-info" (options are alphabetically sorted